eva-accession-release/src/main/java/uk/ac/ebi/eva/accession/release/io/VariantContextWriter.java

```diff
@@ -122,9 +122,12 @@ public void open(ExecutionContext executionContext) throws ItemStreamException {
     private void addContigs(Set<VCFHeaderLine> metaData) {
         try {
             BufferedReader bufferedReader = new BufferedReader(new FileReader(contigsFilePath));
-            String contig;
-            while ((contig = bufferedReader.readLine()) != null) {
-                metaData.add(new VCFHeaderLine("contig", "<ID=" + contig + ">"));
+            String contigLine;
+            while ((contigLine = bufferedReader.readLine()) != null) {
+                String[] contigAndName = contigLine.split(",");
+                String contig = contigAndName[0];
+                String name = contigAndName[1];
+                metaData.add(new VCFHeaderLine("contig", "<ID=" + name + ",Name=\"" + contig + "\">"));
```

Comment:
This is how I have seen it always represented in VCF files out in the wild:
```suggestion
                metaData.add(new VCFHeaderLine("contig", "<ID=" + name + ",accession=\"" + contig + "\">"));
```
Also the name is now in the ID, so the flip makes sense.